packages/dappmanager/test/modules/docker/parseContainerInfo.test.ts

```diff
@@ -398,7 +398,7 @@ describe("modules / docker / parseContainerInfo", function () {
         dependencies: {},
         avatarUrl: "/ipfs/QmQnHxr4YAVdtqzHnsDYvmXizxptSYyaj3YwTjoiLshVwF",
         origin: "/ipfs/QmeBfnwgsNcEmbmxENBWtgkv5YZsAhiaDsoYd7nMTV1wKV",
-        chain: "ethereum",
+        chain: { driver: "ethereum" },
```

Comment:
Why this change? `chain: "ethereum"` must be valid syntax

Reply:
@dapplion fixed on the latest commit!